tool/planet/constants.go

```diff
@@ -307,7 +307,8 @@ const (
 	DefaultPlanetReleaseFile = "/etc/planet-release"
 
 	// AssumeEtcdVersion is the etcd version we assume we're using if we're unable to locate the running version
-	AssumeEtcdVersion = "v2.3.8"
+	// This should be the least available version among the releases we can directly upgrade
+	AssumeEtcdVersion = "v3.3.22"
```

Comment:
Question for @knisbet - do you think it's safe to assume?

Reply:
Are we going to support upgrades from any 5.5, or only from the latest? In the former case, earlier 5.5 versions included etcd v3.3.11 IIRC.